src/pages/user/index.tsx

```diff
@@ -105,31 +150,91 @@ const User: React.FunctionComponent<
               </div>
             </div>
           )}
+          {completeCourseStatus === 'loading' ? (
+            'Loading...'
+          ) : completeCourseStatus === 'error' ? (
+            <span>There was an error fetching completed courses.</span>
+          ) : (
+            <>
+              <div>
+                <h2 className="pb-1 text-xl border-b border-gray-200 dark:border-gray-800">
+                  Completed Courses
+                </h2>
+                <div className="flex flex-wrap gap-4 justify-evenly mt-4">
+                  {completeCourseData.map(
+                    ({
+                      collection,
+                      completed_at,
+                    }: {
+                      collection: CardResource
+                      completed_at: string
+                    }) => {
+                      return (
+                        <div className="flex flex-col justify-between">
+                          <VerticalResourceCard
+                            resource={collection}
+                            location="user profile"
+                            className="text-center w-44 flex flex-col items-center justify-center self-center"
+                          />
+                          <span className="z-10 flex flex-row">
+                            <span>
+                              <BadgeCheckIcon color="green" width="1.5em" />
+                            </span>
+                            <span className="ml-1 h-fit my-auto text-xs text-gray-600 italic font-bold">
```

Comment:
I think I'm doing something funky here with css trying to get the text to align with the svg... `h-fit my-auto` because `align-middle` or `align-baseline` wasn't working